dist/rpm/openQA.spec

```diff
@@ -693,6 +693,7 @@ fi
 %dir %{_unitdir}
 %{_systemdgeneratordir}
 %{_unitdir}/openqa-worker.target
+%{_unitdir}/openqa-worker.slice
```

Comment:
I see that we have `Slice=openqa-worker.slice` in the `[Service]` section of our unit file which is likely also relevant for this new file to be picked up. I'm wondering where `Slice=` is actually documented. I browsed a lot in the different pages of the systemd documentation but couldn't find it. There's not even a mentioning in which section this setting is supposed to go.

Reply:
Thanks, I must have overlooked it.